vizro-core/docs/pages/user-guides/custom-components.md

```diff
@@ -266,6 +266,224 @@ vm.Page.add_type("components", Jumbotron)
     [CustomComponent2]: ../../assets/user_guides/custom_components/customcomponent_2.png
 
 
+## How to use custom components with custom actions
```

Comment:
Could we remove the redundants "how to" across this page to make the menu on the right easier to read?

Reply:
"How to" is used throughout the docs, so removing it here would make custom-components inconsistent with the rest of the docs. @stichbury what do you think? Should we remove the "how to" throughout the docs"?

Reply:
You are correct, it is indeed used in other places - we are actually not really consistent I think as it varies. I may have forgotten what out current stand on this is. @stichbury could you remind us what we decided? I personally much prefer not having lots of redundant "How to" on the side, but I remember that we liked it as section headers...

Reply:
This was my bad, sorry! I added the "How to" back in. I haven't codified the approach but I'll do this somewhere so I remember and can point other authors to it as needed.

So we have a "How-to guides" section and within that, a set of pages (see left hand ToC)

<img width="218" alt="Screenshot 2024-04-03 at 11 51 36" src="https://github.com/mckinsey/vizro/assets/5180475/bb80668e-1e96-4fb8-b7e5-a80476d17e74">

* Guidance 1: Each page should have a "How-to title" such as this one:

<img width="1038" alt="image" src="https://github.com/mckinsey/vizro/assets/5180475/9f8f12cf-bd0b-47b4-a5d0-67115b9a82e4">

* Guidance 2: Subsections on the page should not be "How to..." do different things. The subsections should instead be worded as imperative, so rather than "How to customize pre-defined actions" it should "Customize pre-defined actions". It should also not be gerund form (so don't use "Customizing pre-defined actions".

Purpose of these two "guidances" (note, not rules) is to keep the page clear about what it does (it's a how-to guide) but not overload the right hand ToC with loads of redundant and repetitive text:

<img width="262" alt="image" src="https://github.com/mckinsey/vizro/assets/5180475/52c014f2-e34e-41ee-9eaa-2369e4714c24">


@nadijagraca if you are OK to modify this file, leave me to sort out any other pages where it's currently not following guidance in a separate PR?